examples/bert/bert_auto.json

```diff
@@ -13,15 +13,22 @@
         }
     },
     "systems": {
-        "local_system": {"type": "LocalSystem"}
+        "local_system": {
+            "type": "LocalSystem",
```

Comment:
That is true. The engine defaults to local system with cpu. Should I remove it from all the bert and resnet configs? I kept them so that the user knows how to change the system if needed.

Reply:
Yes, please remove them.

I'm going to update the quick tour to make Olive intro simpler for users.

Reply:
Done. I removed the local system from all the resnet and bert configs. Also updated the default system to initialize with CPU.